python/sports-league-scheduling/src/sports_league_scheduling/demo_data.py

```diff
@@ -0,0 +1,126 @@
+import json
+import random
+from datetime import datetime, time, timedelta
+from typing import List, Callable
+
+from .domain import *
+
+
```

Comment:
You should create a seeded random so the generated demo data is consistent.

```suggestion
random = Random(0)
```